docs/en/connector-v2/Config-Encryption-Decryption.md

```diff
@@ -0,0 +1,189 @@
+# Config  File Encryption And Decryption
+
+## Introduction
+
+In most production environments, sensitive configuration items such as passwords are required to be encrypted and cannot be stored in plain text, SeaTunnel provides a convenient one-stop solution for this.
+
+## How to use
```

Comment:
List the key value pair of encrypt & decrypt

example:
```json
username
password

*username
*password

...
```

Reply:
Done, PTAL